rln/rln.go

```diff
@@ -427,6 +427,14 @@ func (r *RLN) InsertMember(idComm IDCommitment, userMessageLimit uint32) error {
 	return nil
 }
 
+func (r *RLN) InsertRawLeaf(rawLeaf [32]byte) error {
```

Comment:
wdyt about using this type?
```suggestion
func (r *RLN) InsertRawLeaf(rawLeaf MerkleNode) error {
```

Reply:
Indeed! fixed: c07683058f2aff87f6eab319be9a4f9fd1592138